NewHaven/test/ResourcesDriver.cpp

```diff
@@ -2,40 +2,62 @@
 #include <iostream>
 
 int main()
-{
-    HarvestDeck deck;
+{/*
```

Comment:
```
HarvestDeck hDeck;
    HarvestTile* tile;
    ResourceTypes* content;
    std::uint_least8_t deckSize;
    std::uint_least8_t* position;
    ResourceTypes* tContent;
    std::uint_least8_t hDeckSize;
    std::uint_least8_t tPosition;
    for (std::uint_fast8_t i = 0; i < 60; i++)
    {
    tile = deck.draw();
    deckSize = deck.getDeckSize();
    content = tile->getTileContent();
    position = tile->getPosition();
    tile = hDeck.draw();
    hDeckSize = hDeck.getDeckSize();
    tContent = tile->getTileContent();
    tPosition = tile->getPosition();
    std::cout << "We drew a card.\nDeck size: " << static_cast<int>(deckSize) << "\tPosition: "
        << static_cast<int>(position[0]) << ", " << static_cast<int>(position[1]) << '\n';
    std::cout << "We drew a card.\nDeck size: " << static_cast<int>(hDeckSize) << "\tPosition: "
        << static_cast<int>(tPosition) << '\n';
    std::cout << content[0] << '\t' << content[1] << '\n' << content[3] << '\t' << content[2] << "\n\n";
    std::cout << tContent[0] << '\t' << tContent[1] << '\n' << tContent[3] << '\t' << tContent[2] << "\n\n";
    
    tile->rotateTileClockwise();
    delete content;
    content = tile->getTileContent();
    delete tContent;
    tContent = tile->getTileContent();
    std::cout << content[0] << '\t' << content[1] << '\n' << content[3] << '\t' << content[2] << "\n\n";
    std::cout << tContent[0] << '\t' << tContent[1] << '\n' << tContent[3] << '\t' << tContent[2] << "\n\n";
    tile->rotateTileCounterClockwise();
    delete content;
    content = tile->getTileContent();
    delete tContent;
    tContent = tile->getTileContent();
    std::cout << content[0] << '\t' << content[1] << '\n' << content[3] << '\t' << content[2] << "\n\n";
    std::cout << tContent[0] << '\t' << tContent[1] << '\n' << tContent[3] << '\t' << tContent[2] << "\n\n";
    
    delete content;
    delete[] position;
    delete tContent;
    }*/`

```
Why did you comment this out? Isnt it required that the driver shows both the deck for the Tiles and the buildings?

Reply:
I commented that out for the purpose of testing only one thing at a time. For the final submission, it will not be commented.

For the building colors, I think the teacher just worded it weirdly. Either we name their distinctive factor color or type doesn't change a thing in the functionality. I think type is more descriptive.